back-end/index.js

```diff
@@ -16,7 +16,10 @@ app.use('/api/auth', authRoutes);
 
 // protected route
 app.use('/api/games', authMiddleware, gameRoutes)
-app
+
+//test route
+app.use('/api', gameRoutes)
```

Comment:
what is this? @Dz373

Reply:
app.use('/api/games', authMiddleware, gameRoutes) required authorization

Reply:
yes, and that's how it's supposed to be, you don't want your APIs to be public

Reply:
I was using it to test that the APIs worked. I was going to get rid of it after. @vphatfla

Reply:
please remove it now, since you are merging to main branch